File: Application/esc-backend/src/main/java/com/example/escbackend/user/dto/UserDetailsResponse.java
package com.example.escbackend.user.dto;


import com.example.escbackend.common.constants.*;
import lombok.*;
import java.time.OffsetDateTime;
import java.util.UUID;


@Getter
@Builder


public class UserDetailsResponse {
        private UUID id;
        private String phone;
        private String email;
        private UserRole role;
        private UserStatus status;
        private BlackListStatus blacklistStatus;
        private String address;
        private String avatarUrl;
        private String displayName;
        private String businessName;
        private OffsetDateTime createdAt;
}

File: Application/EscrowX/app/src/main/java/mobile/project/escrowx/UserDetailsResponse.kt
package mobile.project.escrowx

import com.google.gson.annotations.SerializedName

data class UserDetailsResponse(
    val id: String,
    val phone: String,
    val email: String,
    val role: String,
    val status: String,
    @SerializedName(value = "displayName", alternate = ["display_name"]) val displayName: String? = null,
    @SerializedName("businessName") val businessName: String? = null,
    @SerializedName("address") val address: String? = null,
    @SerializedName("shopLocation") val shopLocation: String? = null,
    @SerializedName("createdAt") val createdAt: String,
    @SerializedName("updatedAt") val updatedAt: String? = null
)